src/Http/Middleware/LogWithRequestContext.php

```diff
@@ -137,11 +137,15 @@ protected function logOutgoingResponse(Request $request, Response $response): vo
     protected function getUnverifiedToken(Request $request): ?Plain
     {
         if ($bearerToken = $request->bearerToken()) {
-            $parser = new Parser(new JoseEncoder());
-            /** @var Plain $unverifiedToken */
-            $unverifiedToken = $parser->parse($bearerToken);
-
-            return $unverifiedToken;
+            try {
+                $parser = new Parser(new JoseEncoder());
+                /** @var Plain $unverifiedToken */
+                $unverifiedToken = $parser->parse($bearerToken);
+
+                return $unverifiedToken;
+            } catch (Exception $exception) {
+                // token was provided, but not parseable, ignore and continue
```

Comment:
Should we log this? otherwise how would we know from logs this is what happened here?

Reply:
The issue is that just because it's not a JWT token doesn't mean that it's invalid (as per the PAT that's being used). And if the PAT is used, the `Auth::id()` call should work as expected, so the appropriate user should be logged. 

This would only be an issue if an invalid token is passed in, but we have other areas in the code that log when the token is invalid, so this can be seen elsewhere.